### ISSUE TITLE
Errors during downloading metadata for repository 'qubes-contrib-vm-r4.2-current'

### ISSUE THREAD

**Issue by kennethrrosen** [QubesOS/qubes-issues#8599]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release
4.2.0-rc3

### Brief summary
Fedora templates from a 4.2 upgrade-in-place are no longer updating. (both for contrib.qubes-os.org/yum/r4.2/current/vm/**fc37**/repodata/repomd.xml and contrib.qubes-os.org/yum/r4.2/current/vm/**fc38**/repodata/repomd.xml)

### Steps to reproduce
Use Qubes OS Update to update suggested templates.

### Expected behavior
Fedora templates are updated. Debian templates are updated without issues.

### Actual behavior
```
Errors during downloading metadata for repository 'qubes-contrib-vm-r4.2-current':
  - Status code: 404 for https://contrib.qubes-os.org/yum/r4.2/current/vm/fc38/repodata/repomd.xml (IP: 127.0.0.1)
Failed to download metadata for repo 'qubes-contrib-vm-r4.2-current': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```

**Comment by pasyn** [comment]
qubes-contrib is not available yet for 4.2. Use qubes-contrib-testing instead.

Run the following in your AppVM to disable `qubes-contrib` and enable `qubes-contrib-testing`
```
sudo dnf install qubes-repo-contrib
sudo dnf config-manager --set-enabled qubes-contrib-vm-r4.2-current-testing
sudo dnf config-manager --set-disabled qubes-contrib-vm-r4.2-current
```

**Comment by kennethrrosen** [comment]
Can confirm that changing the `/etc/yum.repos.d/qubes-contrib-vm-r4.2.repo` file's "contrib (updates)" repo to `enabled=0` and the "contrib (updates-testing" repo to `enabled=1` solved this issue for me.
